source/frontend/src/features/properties/map/acquisition/AcquisitionView.tsx

```diff
@@ -158,6 +158,8 @@ const getEditTitle = (editFormName: EditFormType) => {
       return 'Updating Acquisition Properties';
     case EditFormType.AGREEMENTS:
       return 'Updating Agreements';
+    case EditFormType.STAKEHOLDERS:
+      return 'Updating Stakeholders';
```

Comment:
nit: I think the pattern for update forms was "Update [XYZ]" - not "Updating [XYZ]"

I checked the uxpin but it doesn't reflect the title change when switching to edit mode so not sure which one should be. I see a mix of "Update..." and "Updating..." in this file